bftengine/tests/testViewChange/testViewChange.cpp

```diff
@@ -147,6 +197,105 @@ void setUpConfiguration_4() {
 
 void cleanupConfiguration_4() { delete sigManager_; }
 
+TEST(testViewchangeSafetyLogic_test, computeRestrictions_two_prepare_certs_for_same_seq_no) {
```

Comment:
@HristoStaykov can you please document this test method, explaining the overall approach, as well as details about the test steps? A bit like what we do for each Apollo test. Thanks!